docs/getting-started/detailed-guide.md

```diff
@@ -0,0 +1,389 @@
+
+    !!! warning "Provisioning times may vary!"
+        Please note that cluster creation time may vary due to provisioning capacity and machine provisioning times of selected hyperscalers.
+
+    After finishing the frontend service reports that the cluster is provisioned.
+
+    ```text
+    2:24PM INF Workflow finished cluster=my-super-cluster module=frontend project=-cloud-bursting
+    ```
+
+12. After finishing provisioning the cluster Claudie creates kubeconfig secret in claudie namespace:
+
+    ```bash
+    kubectl get secrets -l claudie.io/output=kubeconfig
+    ```
+    ```
+    NAME                                  TYPE     DATA   AGE
+    my-super-cluster-6ktx6rb-kubeconfig   Opaque   1      134m
+    ```
+
+    You can recover kubeconfig for your cluster with the following command:
+
+    ```bash
+    kubectl get secrets -n claudie -l claudie.io/output=kubeconfig -o jsonpath='{.items[0].data.kubeconfig}' | base64 -d > my-super-cluster-kubeconfig.yaml
+    ```
+
+    If you want to connect to your machines via SSH, you can recover private SSH key:
+
+    ```bash
+    kubectl get secrets -n claudie -l claudie.io/output=metadata -ojsonpath='{.items[0].data.metadata}' | base64 -d | jq -r .private_key > ~/.ssh/my-super-cluster
+    ```
+
+    To recover public IP of a node to connect to via SSH:
+    ```bash
+    kubectl get secrets -n claudie -l claudie.io/output=metadata -ojsonpath='{.items[0].data.metadata}' | base64 -d | jq -r .node_ips
+    ```
+
+    Each secret created by Claudie has following labels:
+
+    | Key                     | Value                                           |
+    | ----------------------- | ----------------------------------------------- |
+    | `claudie.io/project`    | Name of the project.                            |
+    | `claudie.io/cluster`    | Name of the cluster.                            |
+    | `claudie.io/cluster-id` | ID of the cluster.                              |
+    | `claudie.io/output`     | Output type, either `kubeconfig` or `metadata`. |
+
+
+13. Use your new kubeconfig to see what’s in your new cluster
+
+    ```bash
+    kubectl get pods -A --kubeconfig=my-super-cluster-kubeconfig.yaml
+    ```
+
+14. Let's add a bursting autoscaling node pool in Hetzner cloud. In order to use other hyperscalers, we'll need to add a new provider with appropriate credentials. Open `cloud-bursting.yaml` input manifest again and append the new Hetzner node pool configuration.
+
+    !!! note "Claudie autoscaling"
+        Autoscaler in Claudie is deployed in Claudie management cluster and provisions additional resources remotely at the time of need. For more information check out how [Claudie autoscaling](docs/autoscaling/autoscaling.md) works.
+
+    ```yaml
```

Comment:
I would keep the manifest `name` field here, so the user will be 100% that he has to edit the same manifest file/

Reply:
good idea, added!